sdk/servicebus/service-bus/test/README.md

```diff
@@ -119,14 +119,12 @@ AAD_CLIENT_SECRET=""
 AAD_TENANT_ID=""
 RESOURCE_GROUP=""
 AZURE_SUBSCRIPTION_ID=""
-CLEAN_NAMESPACE="true"
 ```
 
 **Note:**
 * `RESOURCE_GROUP` and `AZURE_SUBSCRIPTION_ID` can be found at your servicebus-namespace in the azure-portal.
```

Comment:
I recall commenting on one of the PRs that `RESOURCE_GROUP` and `AZURE_SUBSCRIPTION_ID` are no longer needed as we dont use the arm-servicebus library anymore.

See https://github.com/Azure/azure-sdk-for-js/pull/6603/files#r362126519

Reply:
Please update this readme file to remove references to these two env variables

Reply:
Ok, looks like the README changes weren't caught in that PR - let's update that in a follow up PR along with the getRule() -> getRuleDetails change